src/Slider.jsx

```diff
@@ -565,14 +566,35 @@ class Rheostat extends React.Component {
     killEvent(ev);
   }
 
+  // Apply user adjustments to position
+  userAdjustPosition(idx, proposedPosition) {
+    const { validateMove } = this.props;
+    let nextPosition = proposedPosition;
+    if (validateMove) {
+      nextPosition = parseFloat(validateMove(idx, proposedPosition));
+
+      if (
+        Number.isNaN(nextPosition)
+        || nextPosition < SliderConstants.PERCENT_EMPTY
+        || nextPosition > SliderConstants.PERCENT_FULL
+      ) {
+        throw new Error('validateMove returned invalid value. Valid values are floats between 0 and 100');
```

Comment:
this should be a `TypeError`. also, let's change this text from "value" to "position".